scripts/ffgscrape.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Downloads card JSON data from https://squadbuilder.fantasyflightgames.com/api/cards/{id}/ and
+ * stores them in ffgcards.json
+ */
+
+var https = require('https');
+var fs = require('fs');
+const { Observable, from, zip } = require('rxjs');
+const { concatMap } = require('rxjs/operators');
+
+let start = 1;
+let end = 491;
+
+let urls = [ ];
+let data = { };
+
+function getCard(url) {
+    console.log("Retrieving " + url);
+    return new Promise((resolve, reject) => {
+        const request = https.get(url, (res) => {
+            let data = "";
+            if (res.status >= 400) {
+                reject(res);
+            } else {
+                res.on('data', (chunk) => {
+                    data += chunk.toString();
+                });
+                res.on('end', () => {
+                    resolve(JSON.parse(data));
+                });
+            }
+        });
+        request.on('error', reject);
+        request.end();
+    });
+}
+
+for (var i = start; i <= end; i = i + 1) {
+    urls.push("https://squadbuilder.fantasyflightgames.com/api/cards/" + i + "/");
+}
+
+let url_obs = from(urls);
+let download_obs = from(urls).pipe(
+    concatMap(
+        (url) => getCard(url)
```

Comment:
Instead of retrieving all cards one by one you could get all cards at once from https://squadbuilder.fantasyflightgames.com/api/cards/